docs/content/concepts/identifier-use/_index.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Identifier Use
+description: Provides details on the scope an duniqueness of dientifiers used within the OSCAL models.
+suppresstopiclist: true
+weight: 50
+toc:
+  enabled: true
+sidenav:
+  focusrenderdepth: 2
+  activerenderdepth: 2
+  inactiverenderdepth: 2
+---
+
+This page reviews important concepts to be aware of when declaring or referencing identifiers in OSCAL models, with in-depth explanations of identifier uniqueness and scope.  
+
+### **Identifier Type** 
+By design, OSCAL supports [*machine-oriented*](/concepts/identifier-use/#machine-oriented) and [*human-oriented*](/concepts/identifier-use/#human-oriented) identifiers. The OSCAL models dictate which are used for different data items.
+
+#### Machine-Oriented
+
+*Machine-oriented* identifiers are implemented as Universally Unique Identifier (UUID) as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122) and are commonly used to (programmatically) identify or refer to unique information within OSCAL models.  Because OSCAL’s *`machine-oriented`* identifiers utilize UUID data types, they can be issued without a central authority, and used across OSCAL instances with low risk of identifier collisions. 
+
+> **NOTE**: While UUIDs are used as machine-oriented identifiers catering to use cases where automated processing is desired, they are not ideal for wildcard matching scenarios.  UUIDs should only be used when exact matches are expected.
+
+
+The [OSCAL Reference Index](/reference/latest/complete/xml-index/#/@uuid) provides a complete listing of UUIDs in OSCAL.  References to these identifiers typically follow a naming convention of the object type followed by “-uuid”.  For example, see the reference index for [location-uuid](/reference/latest/complete/xml-index/#/location-uuid).
```

Comment:
This is a link to the XML index. The JSON index should be referenced as well.

Reply:
added JSON index.  Will be in next commit.

Reply:
valid point.  Updating locally and will include with my next commit

Reply:
This will get crowded with brad's recommendation. It seems we single space between sentences and there is an additional space at "followed by “-uuid”.  For example," between the period and for. I noticed this when re-reading locally.